fetchai/ledger/api/__init__.py

```diff
@@ -38,7 +39,11 @@ def _iterable(value):
 
 
 class LedgerApi:
-    def __init__(self, host, port):
+    def __init__(self, host=None, port=None, server_name=None):
```

Comment:
```suggestion
    def __init__(self, host=None, port=None, network=None):
```

I think the name `network` is better. That endpoint that we have is just a load balancer across a number of nodes